spec/models/accounting/quickbooks/transaction_creator_spec.rb

```diff
@@ -51,23 +83,20 @@
 
   it 'calls create with correct data' do
     expect(generic_service).to receive(:create) do |arg|
-      expect(arg.line_items.count).to eq 2
+      expect(arg.line_items.count).to eq 3
       expect(arg.private_note).to eq memo
       expect(arg.txn_date).to be_nil
 
       list = arg.line_items
-      expect(list.map(&:amount).uniq).to eq [amount]
-      expect(list.map(&:description).uniq).to eq [description]
+      expect(list.map(&:amount)).to eq [line_item_1.amount, line_item_2.amount, line_item_3.amount]
```

Comment:
If you define `line_items` (as an array) instead of separate vars, you can do `line_items.map(&:amount)` here. Similar below.